services/tenant-ui/frontend/src/store/stateStore.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Place for storing global state that is not persisted in the backend.
+ */
+import { defineStore } from 'pinia';
+import { ref } from 'vue';
+
+export const useGlobalStateStore = defineStore('globalState', () => {
+  // state
+  const sidebarOpen: any = ref(null);
```

Comment:
Should this be a true/false for components to know if the sidebar is open? If so I'd make it a boolean type and default it to false.
Or is it a string representing the class name? In which case I'd make it called sidebarOpenClass, type string, and default to whatever the starting class name is (or empty string if it's meant to be indeterminate)

Reply:
We kinda need three states here. That's why I'm using 'null' as a start point. Just plain boolean isn't flexible enough.
We could switch to using a string if you'd like to avoid `null | boolean`.